scripts/legacy_expression_tool.py

```diff
@@ -0,0 +1,394 @@
+            if 'entity_name' in active:
+                if re.search(_regex_legacy_expression, active['entity_name']):
+                    if active['entity_name'] not in found_legacy_expressions:
+                        found_legacy_expressions.append(active['entity_name'])
+                    find_object_occurrences.append({'_id':str(occurrence['_id']), 'expression':active['entity_name'], 'type':'active.entity_name', 'service':occurrence['service'], 'subservice':occurrence['subservice'], 'expressionIndex':found_legacy_expressions.index(active['entity_name'])})
+                    if debug==True:
+                        print ('ocurrence: ' + str(occurrence['_id']) + ' active: ' + str(active['entity_name']))
+                    if translation_legacy_expressions!=[]:
+                        # Do the replacement of the legacy expression
+                        if active['entity_name'] in translation_legacy_expressions[0]:
+                            active['entity_name'] = translation_legacy_expressions[1][translation_legacy_expressions[0].index(active['entity_name'])]
+                            if debug==True:
+                                print(' Replaced expression: "' + active['entity_name'] + '" in object: ' + str(occurrence['_id']))
+                        else:
+                            print('ERROR: Expression not found in translation file: ' + active['entity_name'] + ' in object: ' + str(occurrence['_id']))
+
+            if 'reverse' in active:
+                    if 'expression' in active['reverse']:
+                        if re.search(_regex_legacy_expression, active['reverse']['expression']):
+                            if active['reverse']['expression'] not in found_legacy_expressions:
+                                found_legacy_expressions.append(active['reverse']['expression'])
+                            find_object_occurrences.append({'_id':str(occurrence['_id']), 'expression':active['reverse']['expression'], 'type':'active.reverse.expression', 'service':occurrence['service'], 'subservice':occurrence['subservice'], 'expressionIndex':found_legacy_expressions.index(active['reverse']['expression'])})
+                            if debug==True:
+                                print ('ocurrence: ' + str(occurrence['_id']) + ' active: ' + str(active['reverse']['expression']))
+                            if translation_legacy_expressions!=[]:
+                                # Do the replacement of the legacy expression
+                                if active['reverse']['expression'] in translation_legacy_expressions[0]:
+                                    active['reverse']['expression'] = translation_legacy_expressions[1][translation_legacy_expressions[0].index(active['reverse']['expression'])]
+                                    if debug==True:
+                                        print(' Replaced expression: "' + active['reverse']['expression'] + '" in object: ' + str(occurrence['_id']))
```

Comment:
I'd sugges to use f-string, with use to be easeir to read:

```suggestion
                                        print(f' Replaced expression: {active['reverse']['expression']} in object: {str(occurrence['_id'])}')
```

Reply:
(In general where applicable, not only in this particular line)